src/main/java/com/github/manolo8/darkbot/modules/utils/LegacyFlashPatcher.java

```diff
@@ -83,7 +83,8 @@ private static class FlashInstaller implements Fixer {
                 REG_SVR = Paths.get(System.getenv("WINDIR"), "SysWOW64", "regsvr32");
 
         public boolean needsFix() {
-            return LibSetup.downloadLib("DarkFlash.ocx", FLASH_OCX);
+            String ocxName = "DarkFlash" + (OSUtil.isWindows7OrLess() ? "-W7" : "") + ".ocx";
+            return LibSetup.downloadLib(ocxName, FLASH_OCX);
```

Comment:
this "-W7" logic exists in 2 places, here and in LibUtils#getFlashOcxPath, i think this should reuse the other

Reply:
actually this shouldn't be touched